wgpu-types/src/lib.rs

```diff
@@ -566,6 +566,8 @@ bitflags::bitflags! {
         const VERTEX = 1;
         /// Binding is visible from the fragment shader of a render pipeline.
         const FRAGMENT = 2;
+        /// Binding is visible from the vertex and fragment shaders of a render pipeline.
+        const VERTEX_FRAGMENT = 3;
```

Comment:
let's assign the value as `Self::VERTEX.bits | Self::FRAGMENT.bits`

Reply:
Good call, I have committed this change